docs/TESTS.md

```diff
@@ -1,43 +1,70 @@
-# Running the Tests
+# Exercise testing
 
-1. Open a Command Prompt and get the first exercise:
+Follow the steps below to develop a solution for an exercise and test it locally.
 
-     ```sh
-     exercism download --track ballerina --exercise hello-world
-     ```
+## Download the exercise
 
-2. Change directory to the newly downloaded exercise
+Open a Command Prompt and execute the command below to download the exercise.
 
-     ```sh
-     cd ~/exercism/ballerina
-     ```
+>**Tip:** Replace the value of the `--exercise` property with the name of your preferred exercise.
+
+```sh
+exercism download --track ballerina --exercise hello-world
+```
 
-	The directory location depends on your OS and your configured exercism workspace.
-	See `exercism configure` for your workspace location.
+## Develop your solution
 
-3. Initialize the project prior to executing tests:
+1. Execute the command below to navigate to the directory of the newly downloaded exercise.
+
+     >**Tip:** The directory location in the command below depends on your OS and your configured Exercism workspace. You can find out your workspace location by executing the `exercism configure` command.
 
      ```sh
-     bal init
+     cd ~/exercism/ballerina/hello-world/
      ```
+
+2. Follow the provided instructions and develop a solution for the exercise.
+
+## Test your solution
+
+Some Ballerina exercises will have a single test while others will have multiple tests. The tests can be found in the respective test file of the `/tests` directory inside the root directory of the exercise.
+
+### Test exercise with a single test
+
+Execute the command below to run the test of the exercise.
+
+By default, only the first test is enabled. This is intentional as it allows you to focus on just making that one test pass. 
      
-4. Run the tests:
+```sh
+bal test
+```
 
-     ```sh
-     bal test --offline
-     ```
+### Test exercise with multiple tests
+
+Follow the steps below to test an exercise with multiple tests.
+
+1. Once the first test passes, enable the next test by removing the `enable: false` property shown in the example below from the `@test:Config` annotation of it.
```

Comment:
`shown in the example below from the `@test:Config` annotation of it.` bit too wordy

Reply:
Changed to `Once the first test passes, enable the next test by removing the `enable: false` property from the `@test:Config` annotation of it.`

Reply:
IMO, a simple sentence as follows is good enough. 
```Once the first test passes, enable the next test by removing the `enable: false` property of the `@test:Config` annotation. ```

Reply:
Updated.